gradoop-examples/src/main/java/org/gradoop/utils/converter/deprecated/sources/package-info.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Deprecated sources and necessary classes to read the old csv data.
+ */
+package org.gradoop.utils.converter.deprecated.sources;
```

Comment:
I'm not sure about that package name. I think it should go in `io` instead of `utils`, also it's not a converter, just a Source for an older version of the CSV data.